views/building-hours/format-building-hours.js

```diff
@@ -13,10 +13,16 @@ export function formatBuildingHours(hoursInfo: BuildingInfoType, now: momentT):
   }
 
   let {open, close} = hours
-  let isOpen = isBuildingOpen(hoursInfo, now)
+  let openStatus = isBuildingOpen(hoursInfo, now)
 
-  let openString = open.format(RESULT_FORMAT)
-  let closeString = close.format(RESULT_FORMAT)
+  if (openStatus === 'Almost Closed') {
+    let closingIn = close.diff(now, 'minutes') + 1
 
-  return `${openString} / ${closeString} – ${isOpen}`
+    return `Closing in ${closingIn} minutes`
```

Comment:
I commented on this in #358. I believe when it will show "1 minutes" at some point. We should fix that.